docs/docsite/rst/dev_guide/developing_modules_documenting.rst

```diff
@@ -33,57 +126,66 @@ Include it in your module file like this:
 
 The following fields can be used and are all required unless specified otherwise:
 
-* ``module:``
+:module:
   The name of the module. This must be the same as the filename, without the ``.py`` extension.
```

Comment:
We have the problem of conflating audiences... I think the audience for this doc is modules that will be accepted into ansible/ansible (which others can look at for best practices but can vary as fits their needs.)